integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep38Tests.kt

```diff
@@ -14,9 +14,11 @@ import org.stellar.anchor.util.Sep1Helper.TomlContent
 
 class Sep38Tests(config: TestConfig, toml: TomlContent, jwt: String) {
   private val sep38Client: Sep38Client
+  private val sep38ClientWithoutJwt: Sep38Client
 
   init {
     sep38Client = Sep38Client(toml.getString("ANCHOR_QUOTE_SERVER"), jwt)
+    sep38ClientWithoutJwt = Sep38Client(toml.getString("ANCHOR_QUOTE_SERVER"), "Invalid Token")
```

Comment:
This is only used in `test endpoints does not required valid token when auth is disabled()`. It's better to use local var.